dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -17,6 +17,14 @@ RUN pip install connexion==1.1.15 \
 		six==1.11.0 \
 		urllib3==1.21.1
 
+RUN pip3 install connexion==2.7.0 \
+		setuptools==21.0.0 \
+		grpcio-tools==1.20.0 \
+		certifi==2017.4.17 \
+		python-dateutil==2.6.0 \
+		six==1.11.0 \
+		urllib3==1.21.1
```

Comment:
Suggest to replace tabs with spaces and left-align these lines with `connexion`. Also, please do the same for lines 13-18.

Reply:
Thanks for review. I have taken care now